wpilibj/src/athena/java/edu/wpi/first/wpilibj/IterativeRobot.java

```diff
@@ -307,5 +307,6 @@ public void testPeriodic() {
       System.out.println("Default IterativeRobot.testPeriodic() method... Overload me!");
       m_tmpFirstRun = false;
     }
+    Timer.delay(0.001);
```

Comment:
C++ actually doesn't have any waits in its periodic functions. We should add them to it as well.
https://github.com/wpilibsuite/allwpilib/blob/master/wpilibc/athena/src/IterativeRobot.cpp#L174